dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/utils/PageInfo.java

```diff
@@ -82,10 +82,8 @@ public Integer getTotalPage() {
         if (pageSize == null || pageSize == 0) {
             pageSize = 7;
         }
-        this.totalPage =
-                (this.total % this.pageSize) == 0
-                        ? ((this.total / this.pageSize) == 0 ? 1 : (this.total / this.pageSize))
-                        : (this.total / this.pageSize + 1);
+        
+        this.totalPage = Math.ceil(this.total / this.pageSize);
```

Comment:
```suggestion
        this.totalPage = (int)Math.ceil((double)this.total / (double)this.pageSize);
```

need to do some type conversion